webui/react/src/App.tsx

```diff
@@ -54,12 +56,31 @@ const AppView: React.FC = () => {
   useEffect(() => {
     if (!infoResponse.data) return;
     setInfo({ type: Info.ActionType.Set, value: infoResponse.data });
+
+    // Check to make sure the WebUI version matches the platform version.
+    if (infoResponse.data.version !== process.env.VERSION) {
+      const handleRefresh = (): void => window.location.reload(true);
+      const btn = <Button type="primary" onClick={handleRefresh}>Refresh WebUI</Button>;
+      const message = <div>
+        A new WebUI version <b>{infoResponse.data.version}</b> is available.
+        Please refresh the new WebUI to see the updated changes.
+      </div>;
+      notification.warn({
+        btn,
+        duration: 0,
+        key: 'version-mismatch',
+        message,
+        placement: 'bottomRight',
+      });
+    }
   }, [ infoResponse, setInfo ]);
 
   useEffect(() => {
     setShowSpinner({ opaque: true, type: FullPageSpinner.ActionType.Show });
   }, [ setShowSpinner ]);
 
+  usePolling(fetchInfo, { delay: 1000 });
```

Comment:
suggestion: would this fit better in `AppContexts` ? not directly a context but it fuels info which a app level context.

Reply:
this a great idea!